### ISSUE TITLE
List keyboard shortcuts somewhere

### ISSUE THREAD

**Issue by brunetton** [umap-project/umap#1436]
Two days ago I worked on a project where I had to create around 120 markers, manually. I searched (no very long, I've to admit) for a mention to shortkeys in Wiki pages, but I didn't found anything. Neither Google.
My mouse started to hate me and threatened to bite me :s 

Anyway today I came across `Ctrl+L` totally by chance trying to edit URL and I realized that it's actually a shortcut for creating a line (that makes sense :bulb:). So I "discovered" `Ctrl-P` too. I really would have loved to see those ones in the doc somewhere !

I would like to edit Wiki and add this information. So I'm trying to list all existing shortcuts. I grepped the code and found interesting things in [`umap/static/umap/js/umap.js`](https://github.com/umap-project/umap/blob/8903b8876541928c516a8d858e9bcfd2efb8b5f9/umap/static/umap/js/umap.js#L1884):

- Ctrl+E
- Ctrl+M
- Ctrl+P
- Ctrl+L

Is there any other shortcuts ?

Linked to this I imagine the shortcuts are supposed to be displayed in tooltips, but this isn't the case for me:

![image](https://github.com/umap-project/umap/assets/86322/4c3f7878-b199-4959-ab58-873dd51231c5)

**Comment by yohanboniface** [comment]
Yeah, having those listed inside would help! Thanks anyway for taking the time to add them in the wiki in the meantime.

Here are some I can thing of:

- ctrl-S => save
- ctrl-I => open the import panel
- shift-click on a Marker/Line/Polygon to edit it
- ctrl-shift-click on a Marker/Line/Polygon to edit its layer

We could also add more!

**Comment by brunetton** [comment]
Thanks

> - ctrl-shift-click on a Marker/Line/Polygon to edit its layer

Wow this would have saved me so much time knowing this !

**Comment by yohanboniface** [comment]
> it opens the current layer's properties, but not allows me to change the clicked object's layer

It should open the clicked object layer. To affect another layer to an object: `shift-click` and change in the dropdown menu in the top of the panel.

**Comment by yohanboniface** [comment]
- Ctrl-F => open the search panel
